feature-group-definitions/intersection-observer.yml

```
@@ -0,0 +1,22 @@
+name: IntersectionObserver
```

I think there's a case to be made for either here, but the spec uses the phrase "intersection observer" (not just the interface name) pretty routinely. I think it's more readable with the space and we should do the same. But caniuse does use the interface name, so this is not a blocking suggestion

```suggestion
name: Intersection observer
```

---

I think we should use the phrase form when there's no ambiguity and there's some precedent for it, so I'll accept your suggestion.